<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="bananaLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bananas</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bananaCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="appleLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apples</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="appleCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="breadLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bread</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="breadCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="milkLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="milkCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalCostLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="taxAmountLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax Amount</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="totalCost">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="taxAmount">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
